### DISCUSSION TITLE
Layers

### DISCUSSION THREAD

**Discussion by Finnein** [keiwando/toonsquid#301] [General]
How many layers does ToonSquid support?

**Comment by DinoAwakening** [comment]
I think there's no hard limit. Like, it depends how many layers your device can handle it xd

**Reply by Finnein** [reply]
I have an iPad 2020 (8th gen) with 3 gig ram.

**Reply by DinoAwakening** [reply]
> I have an iPad 2020 (8th gen) with 3 gig ram. 

Yeah... Ram might be your iPad's greatest weakness. Mine has 8gb (iPad Air 5), and gets some drops on performance.

**Comment by keiwando** [comment]
I added an FAQ entry for this:
https://toonsquid.com/handbook/guides/faq/#layer-limit
